_layouts/archives.html

```diff
@@ -5,7 +5,8 @@
 
 <div id="archives" class="pl-xl-2">
 
-{% for post in site.posts %}
+{% assign posts = site.posts | where_exp: "item", "item.hidden != true" %}
```

Comment:
`hidden: true` only hides the post on the homepage pagination, if we want to hide a certain post site-wide, we should use `published: false` instead. So please revert the changes here.

Reply:
Ah ok

Reply:
Actually I'm not sure about this. Using `published` will completly skip generating the page, but what if I want a post that is hidden from the public but can still be accessed if if the URL is known? For this I would use `hidden`, but it still shows up in archives, categories and tags pages which makes it a bit pointless.

Reply:
> what if I want a post that is hidden from the public but can still be accessed if the URL is known?

We could indeed do this by modifying the source code to hide all links to hidden posts on the site, but don't forget that search engines still crawl these hidden posts and display them on the internet. So, what's the point of doing this (`hidden: true`)?

I think the best way to hide a post is to not generate it or encrypt the post content, what do you think?

Reply:
If there are no links to the hidden post then search engines will not find it. It will also need to be hidden from feeds and sitemaps. I suppose it does seem like a pain to do properly since other feed and sitemap plugins don't really support hiding things.